src/sass/components/_about.scss

```diff
@@ -1,5 +1,5 @@
 .about {
-  max-width: $mobile-width;
+  //max-width: $mobile-width;
```

Comment:
just remove the code that you don't use. you're using git, so you have full history of your project if you need it in the future. fix here and everywhere else